packages/uix-core/src/cross-realm-object.ts

```diff
@@ -87,7 +89,9 @@ async function setupApiExchange<T>(
       );
     }),
     tunnel.config.timeout,
-    () => tunnel.destroy()
+    (e) => {
+      tunnel.destroy(e);
+    }
```

Comment:
We now pass errors (usually a timeout error in this case) to the `destroy` method so they can be logged.